common/common.go

```diff
@@ -66,3 +69,33 @@ type Parameters map[string]string
 func (s Parameters) Equals(is Parameters) bool {
 	return reflect.DeepEqual(s, is)
 }
+
+// Write file to temp and atomically move when everything else succeeds.
+// This function is taken from
+//   https://github.com/youtube/vitess/blob/master/go/ioutil2/ioutil.go
+// Copyright 2012, Google Inc. BSD-license
```

Comment:
@nh2 I like your proposal of adding a licenses directories containing the BSD one and point the comment to it.